.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,37 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+    - 'master'
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 1
```

Comment:
Not sure if we show "page last edited" dates, but with a fetch-depth of 1 those would break, as we won't have history.